apps/condo/domains/acquiring/schema/ExportPaymentsService.js

```diff
@@ -75,7 +76,12 @@ const ExportPaymentsService = new GQLCustomSchema('ExportPaymentsService', {
                 const timeZone = normalizeTimeZone(timeZoneFromUser) || DEFAULT_ORGANIZATION_TIMEZONE
                 const formatDate = (date) => dayjs(date).tz(timeZone).format(DATE_FORMAT)
 
-                const objs = await exportPayments(context, { dv: 1, sender, where, sortBy })
+                const objs = await loadListByChunks({
```

Comment:
This should be moved to a subprocess subsequently. If `Payment` table will become too big, this process may cause server slowdown

Reply:
Yep =(

Reply:
This is very dangerous...